src/compiler/diagnosticMessages.json

```
@@ -851,6 +851,10 @@
         "category": "Error",
         "code": 1259
     },
+    "This keyword must not contain an escaped character.": {

```

Minor nit, since `this` is a keyword and the message could be mildly confusing:

```suggestion
    "A keyword must not contain an escaped character.": {
```

Or

```suggestion
    "Keywords must not contain escape characters.": {
```

---

It's funny because Daniel's suggestion was exactly to add the `This` qualifier so the message was _less_ vague sounding and referenced a specific instance, rather than being a general statement.

---

cc @DanielRosenwasser ideas?

---

I would actually prefer

```suggestion
    "Keywords cannot contain escape characters.": {
```

The reason I opted for `This keyword` is because there was concern over the fact that an *IdentifierName* could resemble a keyword.